addons/hr_skills/models/hr_resume.py

```diff
@@ -30,14 +30,17 @@ def create(self, vals_list):
 class ResumeLine(models.Model):
     _name = 'hr.resume.line'
     _description = "Resumé line of an employee"
+    _order = "line_type_id, date_end desc, date_start desc"
 
     employee_id = fields.Many2one('hr.employee', required=True, ondelete='cascade')
     name = fields.Char(required=True)
     date_start = fields.Date(required=True)
     date_end = fields.Date()
     description = fields.Text(string="Description")
     line_type_id = fields.Many2one('hr.resume.line.type', string="Type")
-    sequence = fields.Integer(default=100)
+
+    # Used to apply specific template on a line
+    display_type = fields.Char('Display Type')
```

Comment:
Selection field for types
+
channel_id ---> Courses
survey_id ----> Certification